packages/taco/src/conditions/base/json-api.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+
+import { Condition } from '../condition';
+import {
+  OmitConditionType,
+  returnValueTestSchema,
+} from '../shared';
+
+export const JsonApiConditionType = 'json-api';
+
+export const JsonApiConditionSchema = z.object({
+  conditionType: z.literal(JsonApiConditionType).default(JsonApiConditionType),
+  endpoint: z.string().url(),
+  parameters: z.record(z.string(), z.unknown()).optional(),
```

Comment:
Does `parameters` take any primitive type as a parameter, or just `string` and `object`? Looking at the tests etc., I'm under the impression it's `object` only.
```suggestion
  parameters: z.record(z.object()).optional(),
```

Reply:
I'm not sure I follow. The example lingo from the issue is:
```
    "parameters": {
        "ids": "ethereum",
        "vs_currencies": "usd",
    },
```
so i guess it could be `z.record(z.string(), z.object()).optional(),`